lib/qtrajectory.h

```diff
@@ -68,6 +68,21 @@ using Channel = std::vector<KrausOperator<Gate>>;
 template <typename Gate>
 using NoisyCircuit = std::vector<Channel<Gate>>;
 
+template <typename Gate>
+unsigned count_qubits(const NoisyCircuit<Gate>& ncircuit) {
```

Comment:
+1 to a struct for NoisyCircuit. I think that would also clean up some of the pybind code, since it's a bit finicky about binding STL types to Python objects.

Any preference on the number-of-qubits option name? I hesitate to use `-q` in a library where so many things start with Q - would the current code support a `--num_qubits` option?

Reply:
I would use `-n` for  the number-of-qubits option name.

Reply:
Done.